src/python/bot/fuzzers/libFuzzer/engine.py

```diff
@@ -311,16 +311,19 @@ def fuzz(self, target_path, options, reproducers_dir, max_time):
     })
 
     # Remove fuzzing arguments before merge and dictionary analysis step.
-    arguments = options.arguments[:]
-    libfuzzer.remove_fuzzing_arguments(arguments)
+    merge_arguments = options.arguments[:]
+    libfuzzer.remove_fuzzing_arguments(merge_arguments, is_merge=True)
```

Comment:
my only other alternative is to get rid of `is_merge` parameter and add value profile flag here explicitly, but I lean towards your implementation as it keeps all the code modifying the arguments inside `remove_fuzzing_arguments`

Reply:
yes right, explicit for is_merge feels slightly better.